From e20940f58b998f8eb35e69f73738b88fc267b48f Mon Sep 17 00:00:00 2001
From: Mike Gerdts <mike.gerdts@joyent.com>
Date: Sat, 24 Mar 2018 01:19:41 +0000
Subject: [PATCH] OS-6823 need uefi-csm-rom.bin with fix for OS-6604 Reviewed
 by: Jerry Jelinek <jerry.jelinek@joyent.com> Reviewed by: Patrick Mooney
 <patrick.mooney@joyent.com> Approved by: Patrick Mooney
 <patrick.mooney@joyent.com>

---
 .../generic/usr/share/bhyve/uefi-csm-rom.bin  | Bin 2097152 -> 2097152 bytes
 1 file changed, 0 insertions(+), 0 deletions(-)

diff --git a/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin b/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin
index 02f11ea5..0cbe839c 100755
Binary files a/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin and b/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin differ
-- 
2.21.0

